CMakeLists.txt

```
@@ -4,6 +4,7 @@ project(revpi-tools)
 
 include(GNUInstallDirs)
 
+add_subdirectory(avahi-service)
```

```suggestion
add_subdirectory(avahi-services)
```

I would add an 's' so you reference the meaning to /etc/avahi/services folder. Maybe we add more files here in the future.